tests/tensorflow2/test_estimator.py

```diff
@@ -71,10 +71,11 @@ def test_linear_classifier(out_dir, tf_eager_mode, saveall):
         # vanilla TF 2.2: all = 214, loss = 2, weights = 1, gradients = 0, biases = 12, optimizer variables = 0, metrics = 0, others = 199
         # AWS-TF 2.2: all = 219, loss = 2, weights = 1, gradients = 2, biases = 12, optimizer variables = 5, metrics = 0, others = 197
         # AWS-TF 2.1: all = 226, loss = 2, weights = 1, gradients = 2, biases = 12, optimizer variables = 5, metrics = 0, others = 204
+        # AWS-TF 2.4: all = 229, loss = 2, weights = 1, gradients = 2, biases = 16, optimizer variables = 5, metrics = 0, others = 197
         assert len(tnames) >= 2 + 1 + 12
         assert len(trial.tensor_names(collection=CollectionKeys.LOSSES)) == 2
         assert len(trial.tensor_names(collection=CollectionKeys.WEIGHTS)) == 1
-        assert len(trial.tensor_names(collection=CollectionKeys.BIASES)) == 12
+        assert len(trial.tensor_names(collection=CollectionKeys.BIASES)) >= 12
```

Comment:
If we know the exact number of tensors for TF 2.4, can we do a conditional assertion?

```
assert len == 16 if is_tf_2_4 else assert len == 12
```

Reply:
Yeah, better.